### PR TITLE
Add conda installation information in the README

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
Hi @xEnVrE and @claudiofantacci, thanks to @traversaro the library is available in conda-forge https://anaconda.org/conda-forge/libbayes-filters-lib I opened this PR to add the information in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,13 +38,20 @@ Popular Bayes filters are the **Kalman** [1]-[4] and **particle filters** [5]-[7
...
 The aim of this library is to provide _interfaces_ and _implementations_ for new and existing recursive Bayesian filters!
 
+# Installing with conda
+You can install the binaries with conda. All the dependencies will be automatically installed in the conda environment.
+```bash
+conda install -c conda-forge libbayes-filters-lib
+```
 
-# 🎛 Dependencies
+# Building from sources
```
Comment (xEnVrE) [inline README.md]: Should not we add within this subsection also the part devoted to run tests (as that is only available only after compiling if I am not wrong)?

Thank you

File: README.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ A flexible, modern, cross-platform C++ recursive Bayesian estimation library.
 # Overview
 - [⚠️ About versioning](#️-about-versioning)
 - [📖 Background](#-background)
+- [Installing with conda](#installing-with-conda)
```
Comment (xEnVrE) [inline README.md]: I think we could add a small icon here to adhere with the style of the `README.md`.

Thank you

**Review by xEnVrE — CHANGES_REQUESTED** [review]
Thanks @GiulioRomualdi.

Two very small requests from my side.

**Comment by GiulioRomualdi** [comment]
Hi @xEnVrE I updated the README

**Review by xEnVrE — APPROVED** [review]
LGTM

**Pull request merged**